packages/inference/test/HfInference.test.ts

```diff
@@ -5,6 +5,9 @@ import { jest } from "@jest/globals"; // 3 minute
 import { HfInference } from "../src";
 import { readFileSync } from "fs";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
```

Comment:
Not necessary. Please remove dotenv and rely on passing the token to tests as described in the docs.